versioned_docs/version-0.x.0/kubeslice-cli/installing-kubeslice.mdx

```diff
@@ -0,0 +1,234 @@
+:::
+
+To apply the slice configuration, use the following command:
+```
+kubeslice-cli create sliceConfig -n <project-namespace> -f <slice-configuration-yaml>
+```
+   
+Example
+```
+kubeslice-cli create sliceConfig -n kubeslice-demo -f slice-config.yaml
+```
+   
+Example output
+```
+🏃 Running command: /usr/local/bin/kubectl apply -f slice-config.yaml -n kubeslice-demo
+sliceconfig.controller.kubeslice.io/slice-red created
+
+Successfully Applied Slice Configuration.
+```
+
+## Deploying the Application
+:::info
+If the application is already deployed on a namespace that is onboarded to a slice, then re-deploy the application.
+:::
+
+### Creating a Service Export
+To create a service export, use the following command:
+```
+kubeslice-cli create serviceExportConfig -f <service-export-yaml> -n <application-namespace>
+```
+
+
+### Validating the Service Export
+When an application service runs on one of the worker clusters that are onboarded to a slice, the worker generates a ServiceExport 
+for the application and propagates it to the KubeSlice Controller.
+
+To verify the service export on the controller cluster, use the following command:
+```
+kubeslice-cli get serviceExportConfig -n <project-namespace>
+``` 
+Example
+```
+kubeslice-cli get serviceExportConfig -n kubeslice-demo
+```
+Example Output
+```
+Fetching KubeSlice serviceExportConfig...
+🏃 Running command: /usr/local/bin/kubectl get serviceexportconfigs.controller.kubeslice.io -n kubeslice-demo
+NAME                             AGE
+iperf-server-iperf-kind-ks-w-1   43s
+```
+
+To view the details of the service export configuration, use the following command:
+```
+kubeslice-cli describe serviceExportConfig <resource-name> -n <project-namespace>
+```
+Example
+```
+kubeslice-cli describe serviceExportConfig iperf-server-iperf-kind-ks-w-1 -n kubeslice-demo
+```
+
+The below output shows the ServiceExportConfig for iperf-server application is present on the controller cluster.
+```
+Describe KubeSlice serviceExportConfig...
+🏃 Running command: /usr/local/bin/kubectl describe serviceexportconfigs.controller.kubeslice.io iperf-server-iperf-kind-ks-w-1 -n kubeslice-demo
+Name:         iperf-server-iperf-kind-ks-w-1
+Namespace:    kubeslice-demo
+Labels:       original-slice-name=slice-red
+              service-name=iperf-server
+              service-namespace=iperf
+              worker-cluster=kind-ks-w-1
+Annotations:  <none>
+API Version:  controller.kubeslice.io/v1alpha1
+Kind:         ServiceExportConfig
+Spec:
+  Service Discovery Ports:
+    Name:             tcp
+    Port:             5201
+    Protocol:         TCP
+  Service Name:       iperf-server
+  Service Namespace:  iperf
+  Slice Name:         slice-red
+  Source Cluster:     kind-ks-w-1
+
+```
+
+### Modifying the Service Discovery Configuration
+kubeslice-cli enables you to modify the service discovery parameters. For example, to modify the port on which the service is running edit 
```

Comment:
Add a comma after `running` as in `running, edit...`

Reply:
done